Glottis.h

```diff
@@ -50,7 +50,8 @@ class Glottis
   struct Parameter
   {
     string name;
-    string abbr;
+    string description;
+    //string abbr; //deprecated, P.Krug
```

Comment:
I don't think we need this to stick around. Just remove it, please.

Reply:
done